### ISSUE TITLE
Cannot deploy, at least, marsha with eugene customer in development

### ISSUE THREAD

**Issue by lunika** [openfun/arnold#257]
## Bug Report

**Expected behavior/code**

When I bootstrap the marsha application with eugene customer in development environment it should not fail.

**Actual Behavior**

It fails because secrets are not valid in marsha's vault.

I think the line 19 of file `apps/marsha/templates/app/secret.yml.j2` should be:

```
DJANGO_SENTRY_DSN: "{{ MARSHA_VAULT.DJANGO_SENTRY_DSN | default('') | b64encode }}"
```

I also think that every django applications using sentry can have this issue.

**Steps to Reproduce**

1. `./bin/bootstrap -e "apps_filter=marsha"`

**Environment**
- Arnold version: master
- Platform: linux

**Comment by jmaupetit** [comment]
Yeah it should be fixed as we did for Richie, see https://github.com/openfun/arnold/blob/master/apps/richie/templates/app/secret.yml.j2#L11
